traffic_ops/testing/api/v4/servers_test.go

```diff
@@ -950,6 +954,11 @@ func UpdateTestServers(t *testing.T) {
 		t.Fatalf("Cannot test server type change update; server '%s' had nil type ID", hostName)
 	}
 
+	// Check to verify mtu changed
+	if originalMTU == *respServer.Interfaces[0].MTU {
```

Comment:
Need to check that `respServer.Interfaces` has a length of at least one, and that the 0<sup>th</sup> element has a non-`nil` MTU before dereferencing like this, to avoid a segfault.